src/services/pcn-iptables/src/Iptables.cpp

```diff
@@ -486,4 +486,14 @@ bool Iptables::fibLookupEnabled() {
   }
 
   return fib_lookup_enabled_;
-}
\ No newline at end of file
+}
+
+uint16_t Iptables::interfaceNameToIndex(const std::string &interface_string) {
+  try {
+    auto p = get_port(interface_string);
+    int index = p->index();
+    return index;
+  } catch (std::exception) {
+    throw std::runtime_error("invalid interface name");
```

Comment:
Is this necessary to catch and rethrow the exception?
The problem I see is that we are hiding the real exception, why not just to remove the whole catch block and let the exception go?

Reply:
The only reason is to provide a cleaner message to the user, in case of error, if a wrong interface name is specified.
If you write something like `pcn-iptables -A INPUT -i wrong_iface_name -j ACCEPT`, you will get:
A) `mat::at` (without throw)
B) `invalid interface name` (with throw)

Reply:
After #53 merged, we can remove this try-catch block

Reply:
Done in e24fc86.